README.md

```diff
@@ -16,9 +16,45 @@ limitations under the License.
 
 # <img src="logos/logo.svg" width="180">
 
-## Scalable Data Access Policy Management and Enforcement
+### A Tool for Complex and Scalable Data Access Policy Enforcement
 
-### Project Build
+## Overview
+
+From the client’s perspective, they submit a request to examine data and receive a reference for this data. 
+This reference can then be used to view the data after it has been retrieved and possibly redacted or filtered based on the context of the query and the permissions of the user making the request.
+
+Under the hood, the request involves sending a message that is processed sequentially by a set of the micro-services starting first with Palisade Service.
+The Palisade Service will perform two tasks.
+First it will return a reference, a token back to the client that will uniquely identify this request.
+This token is used throughout the application to tie every aspect of the process to this unique identifier and is later used by the client to retrieve the resources available for this request. 
+The second task for the Palisade Service is to initiate the processing of the request by forwarding it onto the next service in the sequence (in this case the User Service).
+The service will perform its required task and then forward the message onto the next service with the end goal of producing resources ready to be viewed by the client.
+This sequence of steps is completed with the Filtered-Resource Service prepared to provide the processed data.
+In the subsequent request by the client to the Data Service, the service returns, one-by-one upon request over a websocket, metadata for a single resource (e.g. a file) they are authorised to view in some capacity.
+The following diagram shows at a high level the services and their relative relationships with each other. 
+Communications between the services and their related support servers utilise Kafka streaming. 
+
+For a detailed description of the services' workflow see [Appendix I - Palisade Flow Diagram](#appendix-i).  
+For a more technical description of each of the services, follow the links below.
+
+<!--- 
+See doc/services.drawio for the source of this diagram
+--->
+![Service Sequence diagram](doc/services.png)
+
+[Palisade Service](palisade-service/README.md) takes client resource requests and starts the process.  
+[User Service](user-service/README.md) identifies the user in the context of the Palisade service.  
```

Comment:
Although worded not quite right, something like below is more descriptive of the User Service
```
[User Service](user-service/README.md) ensures that the user is valid by checking they exist in a user provider technology.
```

Reply:
Went with the first suggestion for user service